spec/features/hotels/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Hotels Show Page', type: :feature do
+  describe "When I visit a hotel's show page" do
+    before :each do
+      @beverly = Hotel.create!(name: "Beverly Hills Hotel", location: "California")
+      
+      @room_1 = @beverly.rooms.create!(rate: 500, suite: "Madonna Suite")
+      @room_2 = @beverly.rooms.create!(rate: 750, suite: "Judy Garland Suite")
+
+      @guest_1 = Guest.create!(name: "Manila Luzon", nights: 3)
+      @guest_2 = Guest.create!(name: "Latrice Royal", nights: 2)
+      @guest_3 = Guest.create!(name: "Monet Exchange", nights: 5)
+      @guest_4 = Guest.create!(name: "Kasha Davis", nights: 7)
+      @guest_5 = Guest.create!(name: "Anetra", nights: 4)
+      @guest_6 = Guest.create!(name: "Benda La Creme", nights: 1)
+
+      GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_1.id)
+      GuestRoom.create!(guest_id: @guest_3.id, room_id: @room_1.id)
+      GuestRoom.create!(guest_id: @guest_4.id, room_id: @room_1.id)
+      GuestRoom.create!(guest_id: @guest_5.id, room_id: @room_1.id)
+
+      GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_2.id)
+      GuestRoom.create!(guest_id: @guest_2.id, room_id: @room_2.id)
+      GuestRoom.create!(guest_id: @guest_2.id, room_id: @room_2.id)
+      GuestRoom.create!(guest_id: @guest_4.id, room_id: @room_2.id)
+      GuestRoom.create!(guest_id: @guest_3.id, room_id: @room_2.id)
+
+      visit "/hotels/#{@beverly.id}"
+    end
+    
+    it "shows a unique list of all guests that have stayed at this hotel" do
+      within("#guest-list") do
+        expect(page).to have_content(@guest_1.name)
```

Comment:
This is a good test, but to really test for uniqueness with capybara, you'll want to include `,count: 1` as a 2nd argument to the `have_content` method. Ex: 

```
expect(page).to have_content(@guest_1.name, count: 1)
```

This way it should only find one instance of the first argument on the page - if there's more (or less), the test will fail.